test/article.test.js

```diff
@@ -61,168 +102,223 @@ describe('/articles Route', () => {
-          done()
-        })
+                .get('/articles/new')
+                .end((err, res) => {
+                  checkReponse(res)
+                  checkArticleProperties(res)
+                  done()
+                })
     })
 
     it('should be in sorted order from newest to oldest', (done) => {
       chai.request(app)
-        .get('/articles/new')
-        .end((err, res) => {
-          checkDateSorted(res)
-          done()
-        })
+                .get('/articles/new')
+                .end((err, res) => {
+                  checkDateSorted(res)
+                  done()
+                })
     })
   })
 
   describe('GET /articles/new with query', () => {
     it('should have right format response and right properties in articles', (done) => {
       chai.request(app)
-        .get('/articles/new')
-        .query({ lastDate })
-        .end((err, res) => {
-          checkResponse(res)
-          checkArticleProperties(res)
-          done()
-        })
+                .get('/articles/new')
+                .query({ lastDate })
+                .end((err, res) => {
+                  checkReponse(res)
+                  checkArticleProperties(res)
+                  done()
+                })
     })
 
     it('should be in sorted order from newest to oldest', (done) => {
       chai.request(app)
-        .get('/articles/new')
-        .query({ lastDate })
-        .end((err, res) => {
-          checkDateSorted(res)
-          done()
-        })
+                .get('/articles/new')
+                .query({ lastDate })
+                .end((err, res) => {
+                  checkDateSorted(res)
+                  done()
+                })
     })
 
     it('should have date less than lastDate', (done) => {
       chai.request(app)
-        .get('/articles/new')
-        .query({ lastDate })
-        .end((err, res) => {
-          const article = res.body[0]
-          article.publishedAt.should.be.below(lastDate)
-          done()
-        })
+                .get('/articles/new')
+                .query({ lastDate })
+                .end((err, res) => {
+                  const article = res.body[0]
+                  article.publishedAt.should.be.below(lastDate)
+                  done()
+                })
     })
 
     it('should send empty array', (done) => {
       chai.request(app)
-        .get('/articles/new')
-        .query({ lastDate: longAgoDate })
-        .end((err, res) => {
-          res.body.should.be.array
-          res.body.should.be.empty
-          done()
-        })
+                .get('/articles/new')
+                .query({ lastDate: longAgoDate })
+                .end((err, res) => {
+                  res.body.should.be.array
+                  res.body.should.be.empty
+                  done()
+                })
+    })
+  })
+
+  describe('POST /articles/vote', () => {
+    const id = '5846399f9ba87af2501fb035'
+    const fakeId = '5846399f9ba87af2501fb03f'
```

Comment:
I think maybe we should put these in a higher scope describe, since they're being used more than once